src/button/README.md

```diff
@@ -20,8 +56,6 @@ The `vscode-button` is a web component implementation of a [button element](http
 | `type`           | string  | See [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attributes). |
 | `value`          | string  | See [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attributes). |
 
-## Usage
-
 ### Basic Usage
```

Comment:
You used "Default" as the title in the Badge docs, could you choose one or the other (I have no preference) and apply it across all component docs?